core/render/vi/__init__.py

```diff
@@ -77,56 +67,45 @@ def getStructure(adminTree, module):
     return json.dumps(res or None, cls=CustomJsonEncoder)
 
 
+@exposed
 def setLanguage(lang, skey):
     if not securitykey.validate(skey):
         return
```

Comment:
Btw: Why does this not raise an exception (`PreconditionFailed`) as everywhere else?

Reply:
tbd

Reply:
fixed